code/socialDistribution/templates/tagtemplates/author.html

```diff
@@ -1,45 +1,64 @@
 {% load static %}
-<div class="author-tile card shadow bg-white m-5 h-100">
+<div class="author-tile card shadow bg-white m-2 h-100 {% if not comment_card %} m-5 {% endif %}">
```

Comment:
Does it really make cohesive sense to use the author card for comments as well? To me, it feels like comment tiles and author tiles have a different purpose and meaning. Just thinking about code maintainability. We can always refactor later too

Reply:
Was easy to reuse the card but I will create a comment-card if it grows too big.